stubs/protobuf/METADATA.toml

```diff
@@ -1,2 +1,2 @@
-version = "3.20.*"
+version = "4.21.*"
```

Comment:
This appears to have created a 4.21.0.0 for whatever reason. It's not strictly problematic, but it's a bit weird to have 

https://pypi.org/project/types-protobuf/4.21.0.0/
and
https://pypi.org/project/protobuf/4.21.9/

Reply:
just a bit awkward to have a 4 section version for the types and a 3 section version for the regular one.

Reply:
I made this change to stub_uploader a month or two ago. The reason is to more clearly distinguish type versions from upstream versions and to demonstrate the intended version compatibility range of `4.21.*`.
E.g., if we bumped the third number you may naively think that types-protobuf 4.21.1 or 4.21.2 corresponded to those versions of protobuf.

Reply:
Logic is here: https://github.com/typeshed-internal/stub_uploader/blob/8a5c11be0539b5a5170eb426cfd1a066996fbaa9/stub_uploader/get_version.py#L51

Reply:
ah intentional. Cool. Didn't realize. Thanks for the explanation